templates/hyper_parameter_tuning_job/sdk_delete_pre_build_request.go.tpl

```diff
@@ -0,0 +1,6 @@
+    // Call StopHyperparameterTuningJob only if the job is InProgress, otherwise just return nil to mark the 
```

Comment:
femto-nit.. indentation off on these first lines in your templates. It's corrected by `go fmt` in the end generated code, but if you want to fix it up now, cool. :)

Reply:
+1
I copy-pasted the code from generated sdk.go and the only diff was an extra whitespace. didn't see an indentation diff
pushed the change